partner_ref_unique/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2016 Antiun Ingenieria S.L. - Antonio Espinosa
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Partner unique reference",
+    "summary": "Add an unique constraint to partner ref field",
+    "version": "11.0.1.0.0",
+    "category": "Customer Relationship Management",
+    "website": "http://www.tecnativa.com",
```

Comment:
Update website as commented in https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10